src/incremental/tests/test_update.py

```diff
@@ -71,7 +71,7 @@ def test_create(self):
 """
 
 # This file is auto-generated! Do not edit!
-# Use `python -m incremental.update inctestpkg` to change this file.
+# Use `incremental` to change this file.
```

Comment:
I dropped the package name here because I plan to make it optional in #111 and didn't want to ask someone to review all these test cases twice.